src/main/java/io/codemc/bot/commands/CmdCodeMC.java

```diff
@@ -0,0 +1,487 @@
+            this.help = "Fetch information about the Nexus Service.";
+
+            this.options = List.of(
+                    new OptionData(OptionType.STRING, "user", "The user that owns the repository.").setRequired(true)
+            );
+        }
+
+        @Override
+        public void withModalReply(SlashCommandEvent event) {
+        }
+
+        @Override
+        public void withHookReply(InteractionHook hook, SlashCommandEvent event, Guild guild, Member member) {
+            String user = event.getOption("user", null, OptionMapping::getAsString);
+
+            if (user == null || user.isEmpty()) {
+                CommandUtil.EmbedReply.from(hook).error("Invalid Username provided!").send();
+                return;
+            }
+
+            String nexusUrl = bot.getConfigHandler().getString("nexus", "url");
+            String repository = user.toLowerCase();
+
+            CompletableFuture<JsonObject> nexus = new CompletableFuture<>();
+            nexus.whenCompleteAsync((info, ex) -> {
+                if (ex != null) {
+                    CommandUtil.EmbedReply.from(hook).error("Failed to fetch Nexus Repository Info!").send();
+                    LOGGER.error("Failed to fetch Nexus Repository Info for '{}'!", user, ex);
+                    return;
+                }
+
+                if (info == null) {
+                    CommandUtil.EmbedReply.from(hook).error("Failed to fetch Nexus Repository Info!").send();
+                    return;
+                }
+
+                String format = ((JsonPrimitive) info.get("format")).getContent();
+                String type = ((JsonPrimitive) info.get("type")).getContent();
+
+                MessageEmbed embed = CommandUtil.getEmbed()
+                        .setTitle(user, nexusUrl + "/#browse/browse:" + repository)
+                        .setDescription("Information about the Nexus Repository.")
+                        .addField("Format", format, true)
+                        .addField("Type", type, true)
+                        .setTimestamp(Instant.now())
+                        .build();
+
+                hook.sendMessageEmbeds(embed).queue();
+            });
+            NexusAPI.getNexusRepository(repository, JavaContinuation.create(nexus));
+        }
+    }
+
+    private static class Remove extends BotCommand {
+
+        public Remove(CodeMCBot bot) {
+            super(bot);
+
+            this.name = "remove";
+            this.help = "Remove a user from a CodeMC Service.";
+
+            this.allowedRoles = bot.getConfigHandler().getLongList("allowed_roles", "commands", "codemc");
+
+            this.options = List.of(
+                    new OptionData(OptionType.STRING, "username", "The Jenkins/Nexus username of the user.").setRequired(true),
+                    new OptionData(OptionType.USER, "discord", "The discord user that is having their status revoked. Leave blank if the user is no longer in the server.")

```

Comment:
Description is too long (Can't be more than 100 characters).
My change:
```suggestion
                    new OptionData(OptionType.USER, "discord", "ID of Discord user having their status revoked. Leave blank for users no longer in the server.")

```